generators/ruby/sdk/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.0.2] - 2024-02-01
+
+- Improvement: Support output mode block configuration
+- Improvement: Consolidate imports for easier gem usage
```

Comment:
Worth including a code snippet as here imo just for team visibility / getting in the habit. in particular, what was the import before and whats it after